File: reactive-demo-backend-flux/src/main/java/dev/divezone/demo/reactive/flux/config/WebConfig.java
package dev.divezone.demo.reactive.flux.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

import dev.divezone.demo.reactive.flux.setup.Setup;
import dev.divezone.demo.reactive.flux.web.AirlineController;

@Configuration
@ComponentScan(basePackageClasses = {Setup.class, AirlineController.class})
public class WebConfig {
}

File: reactive-demo-backend-kotlin/src/main/kotlin/dev/divezone/demo/reactive/config/WebConfig.kt
package dev.divezone.demo.reactive.config

import dev.divezone.demo.reactive.setup.Setup
import dev.divezone.demo.reactive.web.AirlineController
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

@Configuration
@ComponentScan(basePackageClasses = [Setup::class, AirlineController::class])
class WebConfig